core/instrumentation.ts

```diff
@@ -0,0 +1,16 @@
+import { OTLPHttpProtoTraceExporter, registerOTel } from '@vercel/otel';
+
+const host = process.env.LIGHTSTEP_COLLECTOR_HOST || 'ingest.lightstep.com';
```

Comment:
May be good to keep these names generic to not tie this to Lightstep too heavily.

Reply:
I wanted to be a little specific as I've only tested this with lightstep but what you've said also makes sense. As I'll probably add some documentation on how to use it, we can mention it there I suppose to specify on what was tested.